### ISSUE TITLE
My ags does not start[Issue]

### ISSUE THREAD

**Issue by gabdepa** [end-4/dots-hyprland#990]
- I have read the Usage and Troubleshooting pages of the wiki: https://end-4.github.io/dots-hyprland-wiki/en/i-i/04troubleshooting/
- I have made sure that both my config and system packages are up to date
- Linux distro: Arch Linux

---
## AGS won't start

My ags won't start on Hyprland run, when i try to load manually this is what i get:
<details>
<summary>Logs</summary>

```plain
[grpp@R2-D2][~] $> ags run .config/ags/init.js 

(gjs:2494): Gjs-WARNING **: 19:21:41.520: JS ERROR: ImportError: Unable to load file from: resource:///com/github/Aylur/ags/app.js (O recurso em “/com/github/Aylur/ags/app.js” não existe)


(gjs:2494): Gjs-CRITICAL **: 19:21:41.521: Failed to resolve imports for module: 'file:///run/user/1000/ags.js'
error: exit status 1 -->
```

</details>

**Comment by xu8533** [comment]
same to me

**Comment by celesrenata** [comment]
**TLDR**: somebody updated AUR to point to v2 without considering all the breaking changes.

modify your PKGBUILD to point to:
`pkgver=1.8.2`


I've been looking into what it would take to migrate these dots to a working state with AGSv2/Astral, which is what is causing your error (you have v2 installed). I've gotten pretty far, but it still won't start yet!
